scripts/rebuild-nimble

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Used to rebuild nimble if not already build
+
+
+# Windows detection
+if uname | grep -qiE "mingw|msys"; then
+  EXE_SUFFIX=".exe"
+  OS="windows"
+  BUILD_COMMAND="./compile_nimble.bat"
+else
+  EXE_SUFFIX=""
+  OS="linux"
+  BUILD_COMMAND="sh compile_nimble.sh"
+fi
+
+NIMBLE_CACHE_DIR=~/.cache/nim/nim-workspace/
+NIMBLE=$NIMBLE_CACHE_DIR/nimble_$NIMBLE_COMMIT$EXE_SUFFIX
+
+if [ ! -f $NIMBLE ]; then
+    mkdir -p $NIMBLE_CACHE_DIR
+    pushd ./scripts/nimble-$OS/
+    $BUILD_COMMAND
+    popd
+    cp ./scripts/nimble-$OS/nimble $NIMBLE
+    mkdir bin -p
+    rm ./bin/nimble$EXE_SUFFIX
+    ln -s $NIMBLE ./bin/nimble$EXE_SUFFIX
```

Comment:
At the moment, this script won't be able to switch back to a previously used Nimble version (e.g. for example, we may discover an issue in a Nimble upgrade and then decide to rollback to the previous version).

The solution may be another if that checks whether the link is up to date (or creating the link unconditionally outside of this if)

Reply:
I guess creating the link unconditionally should be good enough.